### PR TITLE
add vifm

### PR DESCRIPTION
**Pull request by Rex-Diego**
0.8.2

### CODE REVIEW

**Comment by ocefpaf** [comment]
> Will this be merged into conda-forge packages?

Sure. We only need to review it first.

> I'm new to conda-forge, just following the simple document and write this recipe.

Perfect!

> I'm doing it right to pull request here?

Yep.

Please see http://conda-forge.github.io/docs/ciservices.html#circleci-linux to enable `CircleCI` testing on your branch and hence the testing hook here so we can see the Linux builds.

File: recipes/build.sh [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+./configure --prefix=$PREFIX \
+            --disable-dependency-tracking \
+            $OPTS
+
+
```
Comment (ocefpaf) [inline recipes/build.sh]: Is there a `make check` we can run here?

File: recipes/example/meta.yaml [hunk 2]
```diff
@@ -1,72 +0,0 @@
-# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
```
Comment (ocefpaf) [inline recipes/example/meta.yaml]: Please do not remove our example recipe! Other will need it :wink:

File: recipes/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,53 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
```
Comment (ocefpaf) [inline recipes/meta.yaml]: But please remove the example comments hen you've finalized your recipe.

File: recipes/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - ncurses
+
+  run:
+    - libgcc
```
Comment (ocefpaf) [inline recipes/meta.yaml]: If you don't need `gcc` library at build time you probably don't need `libgcc` at run time.

File: recipes/build.sh (hunk 1 above)
Comment (Rex-Diego) [inline recipes/build.sh]: Yes, I will add it and test.

File: recipes/example/meta.yaml (hunk 2 above)
Comment (Rex-Diego) [inline recipes/example/meta.yaml]: Sorry, I have no idea about that. Now, I'm bring it back!

File: recipes/meta.yaml (hunk 3 above)
Comment (Rex-Diego) [inline recipes/meta.yaml]: It's already cleaned up now.

File: recipes/meta.yaml (hunk 4 above)
Comment (Rex-Diego) [inline recipes/meta.yaml]: That's true, I've removed it.

File: recipes/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+  run:
+    - ncurses
+
+test:
+  commands:
+    - vifm --help
+
+about:
+  home: https://vifm.info/
+  license: GPL-2.0
```
Comment (scopatz) [inline recipes/meta.yaml]: Please add a `license_file` entry, since this is GPL'd the license is required.

**Comment by scopatz** [comment]
Hi @Rex-Diego, thanks for putting this in. Everything here needs to be in a `recipies/vifm` directory, not just in `recipes`. Please see some other PRs for examples. 

Also, please enable CircleCI for your fork of staged recipes: http://conda-forge.github.io/docs/ciservices.html#circleci-linux

**Comment by Rex-Diego** [comment]
@scopatz I will add circle CI.

But I didn't find any example that put the "meta.yml" and "build.sh" files in recipes/name-of-the-package path.  i.e. Some passed PR:[https://github.com/pstjohn/staged-recipes](https://github.com/pstjohn/staged-recipes)

So, as I didn't find an example for reference, shall I just make a directory vifm in the folder recipes and put "meta.yml" and "build.sh" in recipes/vifm?

File: recipes/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+  license_file: https://github.com/vifm/vifm/blob/master/COPYING
```
Comment (scopatz) [inline recipes/meta.yaml]: I think this needs to be a file in the source tree and not a URL

File: recipes/meta.yaml [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+  summary: 'vifm 0.8.2'
```
Comment (scopatz) [inline recipes/meta.yaml]: A better description is desirable.

**Comment by scopatz** [comment]
Hi @Rex-Diego, basically all other PRs on this repo have that structure, for example:https://github.com/conda-forge/staged-recipes/pull/2162/files  You probably shouldn't look at a fork, just other PRs.  Also note that feedstocks (which hold a single recipe) have a slightly different structure because of this.

And yes, please place them in that dir.

File: recipes/meta.yaml (hunk 6 above)
Comment (Rex-Diego) [inline recipes/meta.yaml]: Yes, you're right.
Now, it's correct.

**Comment by scopatz** [comment]
Thanks @Rex-Diego

**Comment by jakirkham** [comment]
FYI it appears CircleCI is failing on your feedstock, but you will need to accept the invitation to conda-forge in order to fix it.

**Pull request merged**